### PR TITLE
Fix missing render path error

### PR DESCRIPTION
**Pull request by marchersimon**
I tried fixing another uncaught error, when running `tldr --render` without a path. Insead of getting
```
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string or an instance of Buffer or URL. Received type boolean (true)
    at readFile (node:fs:351:10)
    at go$readFile (/usr/lib/node_modules/tldr/node_modules/graceful-fs/graceful-fs.js:118:14)
    at Object.readFile (/usr/lib/node_modules/tldr/node_modules/graceful-fs/graceful-fs.js:115:12)
    at /usr/lib/node_modules/tldr/node_modules/universalify/index.js:10:14
    at new Promise (<anonymous>)
    at Object.readFile (/usr/lib/node_modules/tldr/node_modules/universalify/index.js:7:14)
    at Tldr.render (/usr/lib/node_modules/tldr/lib/tldr.js:76:15)
    at Object.<anonymous> (/usr/lib/node_modules/tldr/bin/tldr:104:12)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10) {
  code: 'ERR_INVALID_ARG_TYPE'
}
```
I now get 
```
/home/simon/Desktop/tldr-node-client/bin/tldr:112
    throw new MissingRenderPathError();
    ^

MissingRenderPathError: Option --render needs an argument.
    at Object.<anonymous> (/home/simon/Desktop/tldr-node-client/bin/tldr:112:11)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Function.Module._load (node:internal/modules/cjs/loader:828:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  code: 4
}
```
I know it's not much, but maybe a start. Feel free to close the PR if it isn't helpful.

### CODE REVIEW

File: bin/tldr [hunk 1]
```diff
@@ -108,6 +108,9 @@ if (program.list) {
       return tldr.updateIndex();
     });
 } else if (program.render) {
+  if (typeof program.render !== 'string') {
+    throw new MissingRenderPathError();
+  }
   p = tldr.render(program.render);
```
Comment (vladimyr) [inline bin/tldr:114]: Throw error inside render method instead 💡

**Review by vladimyr — COMMENTED** [review]
> I know it's not much, but maybe a start. Feel free to close the PR if it isn't helpful.

Quality matters more than quantity 😉 and looking from that perspective this is certainly a good effort 👍

File: lib/errors.js [hunk 2]
```diff
@@ -29,16 +29,27 @@ class MissingPageError extends TldrError {
...
   }
 }
 
+class MissingRenderPathError extends TldrError {
+  constructor() {
+    super(trim`
+Option --render needs an argument.
+    `);
+    this.name = 'MissingRenderPathError';
+    this.code = 4;
```
Comment (vladimyr) [inline lib/errors.js:44]:
```suggestion
    // eslint-disable-next-line no-magic-numbers
    this.code = 4;
```

**Comment by vladimyr** [comment]
LGTM I believe it is ready for prime time by removing draft status 😉

File: lib/errors.js [hunk 3]
```diff
@@ -29,16 +29,28 @@ class MissingPageError extends TldrError {
...
     // eslint-disable-next-line no-magic-numbers
     this.code = 3;
   }
 }
 
+class MissingRenderPathError extends TldrError {
+  constructor() {
+    super(trim`
+Option --render needs an argument.
+    `);
```
Comment (bl-ue) [inline lib/errors.js:42]: Any reason not to do this, @vladimyr?
```suggestion
    super('Option --render needs an argument.');
```
Just for consistency?  It's not a problem; I'm just curious.

File: lib/errors.js [hunk 4]
```diff
@@ -29,16 +29,28 @@ class MissingPageError extends TldrError {
 Page not found.
 If you want to contribute it, feel free to send a pull request to: ${repo}
     `);
-    this.name = 'MissingPageErrror';
```
Comment (bl-ue) [inline lib/errors.js:32]: 😆

**Review by bl-ue — APPROVED** [review]

File: lib/errors.js (hunk 3 above)
Comment (vladimyr) [reply lib/errors.js:42]: Honestly, I failed to spot it. Sure let's do that 👍

**Comment by marchersimon** [comment]
Do you think this can already be merged? The output is still pretty long:
```
Desktop/tldr-node-client/lib/tldr.js:77
      throw new MissingRenderPathError();
      ^

MissingRenderPathError: Option --render needs an argument.
    at Tldr.render (/home/simon/Desktop/tldr-node-client/lib/tldr.js:77:13)
    at Object.<anonymous> (/home/simon/Desktop/tldr-node-client/bin/tldr:111:12)
    at Module._compile (node:internal/modules/cjs/loader:1108:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)
    at Module.load (node:internal/modules/cjs/loader:988:32)
    at Function.Module._load (node:internal/modules/cjs/loader:828:14)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)
    at node:internal/main/run_main_module:17:47 {
  code: 4
}
```

**Review by vladimyr — APPROVED** [review]

**Comment by marchersimon** [comment]
Any updates on this?

**Comment by agnivade** [comment]
You are welcome to merge this by yourself, or let me know, I am happy to do it.

**Pull request merged**